src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx

```diff
@@ -43,7 +42,7 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
     const supersetServiceMock: any = jest.fn(async () => []);
     const props = {
       currentGoal: fixtures.goal3.goal_id,
-      goals: [fixtures.goal3],
+      goals: [fixtures.goal3 as goalDucks.Goal],
```

Comment:
Is this because they're being typed as `Goal`s in the fixtures file?

Reply:
Its because for some reason Typescript cannot infer what type fixtures.goal3 is.  I decided to do this after trying and failing to change the fixture.